### PR TITLE
Subclass *Attributes for testing non-production methods #5479

### LINKED ISSUE #5479: Subclass *Attributes for testing non-production methods
As suggested by @wkurniawan07 in #5446, subclasses of classes currently containing non-production method can be created as inner classes in `*AttributesTest`. The non-production methods will only be on the subclasses so we can ensure that they won't be used elsewhere.

### PR DESCRIPTION
**Pull request by wkurniawan07**
Fixes #5479 

Only one problem: I don't know how to name the subclasses :p
(currently `*AttributesExt` means `*Attributes` extended)

### CODE REVIEW

File: src/test/java/teammates/test/cases/storage/FeedbackQuestionsDbTest.java [hunk 1]
```diff
@@ -23,6 +23,18 @@
 
 public class FeedbackQuestionsDbTest extends BaseComponentTestCase {
     private static final FeedbackQuestionsDb fqDb = new FeedbackQuestionsDb();
+    
+    private class FeedbackQuestionAttributesExt extends FeedbackQuestionAttributes {
```
Comment (kanghj) [inline src/test/java/teammates/test/cases/storage/FeedbackQuestionsDbTest.java]: Name this as `FeedbackQuestionAttributesWithModifiableTimestamps` / `ModifiableTimestampsFeedbackQuestionAttributes`? 

Can't really think of a good name for this too.

Comment (wkurniawan07) [inline src/test/java/teammates/test/cases/storage/FeedbackQuestionsDbTest.java]: I guess `*WithModifiableTimestamp` is good enough.

File: src/test/java/teammates/test/cases/storage/FeedbackQuestionsDbTest.java [hunk 2]
```diff
@@ -24,6 +24,18 @@
 public class FeedbackQuestionsDbTest extends BaseComponentTestCase {
     private static final FeedbackQuestionsDb fqDb = new FeedbackQuestionsDb();
 
+    private class FeedbackQuestionAttributesWithModifiableAttributes extends FeedbackQuestionAttributes {
```
Comment (kanghj) [inline src/test/java/teammates/test/cases/storage/FeedbackQuestionsDbTest.java]: Should this be `FeedbackQuestionAttributesWithModifiableTimestamp`?

Comment (wkurniawan07) [inline src/test/java/teammates/test/cases/storage/FeedbackQuestionsDbTest.java]: Oops.
Anyway looks like Travis is a bit drunk, they're pulling the old `.yml` file with the two separate configs even after syncing. I might have to close this PR and open a new one.

Comment (wkurniawan07) [inline src/test/java/teammates/test/cases/storage/FeedbackQuestionsDbTest.java]: Yup, rebasing works. Can review again.

**Comment by wkurniawan07** [comment]
@kanghj I'm seeing something funny here though. Why is a `*Attributes` tested in a `*DbTest`?

**Comment by kanghj** [comment]
Let's shift the affected tests to `*AttributesTest` before merging then. I guess the original attempt to implement for this was to use the Db to add the default timestamps, hence `*DbTest` was appropriate.

**Comment by wkurniawan07** [comment]
@kanghj done, one last review (and apply the Approved label) perhaps?

**Pull request merged**